README.md

```diff
@@ -6,6 +6,8 @@
 
 Playwright is a Node.js library to automate [Chromium](https://www.chromium.org/Home), [Firefox](https://www.mozilla.org/en-US/firefox/new/) and [WebKit](https://webkit.org/) with a single API. Playwright is built to enable cross-browser web automation that is **ever-green**, **capable**, **reliable** and **fast**.
 
+> Looking for **Playwright for .NET?** Head over to https://github.com/microsoft/playwright-dotnet for the latest info and release!
```

Comment:
We don't actually want to send customers to the github readme, most of the useful content is here: https://playwright.dev/dotnet/docs/intro/. And we link to it from https://playwright.dev, in every page toolbar. I wonder if that is sufficient!

Reply:
The reality is the readme exists, has a LOT of info on it, and would be helpful to the .NET developers as a pointer to the right repo where their Playright source/info is at as well.  I understand the marketing site playwright.dev but as long as the README's exist on this main repo (which playwright.dev links to as well), it seems a nominal addition to add clarity to the visitor of this repo.

Reply:
I agree with Tim - our page already links to all the languages, as it should. Our readme _should_ also at least mention the languages we support - but, as per my suggestion here, I'd want to mention all the languages. If/when this PR merges, I'll also open PRs on other languages to cross-link.

Reply:
The problem is that we don't want to send anyone to https://github.com/microsoft/playwright-dotnet. We don't really maintain it, don't update it with latest and greatest snippets, etc. etc.

> The reality is the readme exists, has a LOT of info on it

So far it is two snippets and a paragraph of text, so I'm trying to make sure it does not get a lot of information.

> I understand the marketing site playwright.dev

That's exactly the misconception I'm trying to mitigate - playwright.dev contains 20,000 lines of markdown documentation for Playwright, per language, with guides, examples, etc. Repo does not have any of that information, only playwright.dev does. If 140 is a LOT I wonder what you say about 20K! I'd like to make sure people don't think README.md is a viable replacement for playwright.dev.